app/code/Magento/InventoryInStorePickupAdminUi/Plugin/Sales/Order/View/ShippingAddress/HideEditLink.php

```diff
@@ -44,7 +44,7 @@ public function __construct(
     public function afterGetAddressEditLink(Info $subject, string $result, Address $address): string
     {
         if ($address->getAddressType() === self::TYPE_SHIPPING) {
-            if ($this->isStorePickupOrder->execute((int)$address->getOrder()->getEntityId())) {
+            if ($this->isStorePickupOrder->execute([(int)$address->getOrder()->getEntityId()])) {
```

Comment:
How would this work with the `OperationResult`? Looks like it would always evaluate to `true`.